scripts/generate-docs.js

```diff
@@ -7,11 +7,12 @@ echo('Appending docs to README.md');
 cd(__dirname + '/..');
 
 // Extract docs from shell.js
-var docs = grep('//@', 'shell.js');
+var docs = grep('^//@', 'shell.js');
```

Comment:
This makes the regex a bit more specific than what it was before. This lets us have comments like what we see in shell.js line 30:

```javascript
// The //@include includes the docs for that command
```

Otherwise, this would attempt to include docs from a file named `includes the docs for that command.js` (which obviously doesn't exist).